src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -223,21 +223,14 @@
     private File pedigreeFile = null;
 
     private FamilyLikelihoods famUtils;
-    private SampleDB sampleDB = null;
 
     private VariantContextWriter vcfWriter;
 
     @Override
     public void onTraversalStart() {
-        final VariantContextWriterBuilder builder = new VariantContextWriterBuilder().setOutputFile(out).setOutputFileType(VariantContextWriterBuilder.OutputType.VCF);
-        if (hasReference()){
```

Comment:
I already wasn't happy with the reference checking in this tool and then you took out more of it?